experimental/PropertyDDS/packages/property-proxy/src/componentArray.ts

```diff
@@ -128,49 +133,43 @@ class ComponentArray extends Array {
     /**
      * @inheritdoc
      */
-    lastIndexOf(searchElement, fromIndex) {
-        // Only handle non-primitive cases,
-        // primitive cases can be handled implicitly (see componentArrayProxyHandler.js)
-
-        if (PropertyFactory.instanceOf(!searchElement, 'BaseProperty')) {
```

Comment:
this branch should just never be taken !searchElement coearces it to a boolean so will either be false or true(if false was the provided element) neither of the cases will ever be an instance of `BaseProperty` therefor this if branch will never be taken